README.md

```diff
@@ -2,15 +2,13 @@
 The Paketo Buildpack for Dynatrace is a Cloud Native Buildpack that contributes the Dynatrace OneAgent and configures it to connect to the service.
 
 ## Behavior
-This buildpack will participate if one the following conditions are met
+This buildpack will participate if exactly one the following conditions are met
```

Comment:
```suggestion
This buildpack will participate if either of the following conditions is met
```

From looking at `IsDynatraceBinding`, I think this is more accurate. It's going to return true if either of these statements is true.

From looking at the test cases, I think the intent of saying "exactly" is to indicate that you may only have one bound service at a time. Perhaps we could clarify that with another note?

```
**Note**:  While a single binding may match both conditions, you may *not* have multiple bindings that match the conditions above. Multiple bindings are not supported.
```

Reply:
I have added the clarifying note.